src/NosCore.GameObject/Character.cs

```diff
@@ -443,15 +448,15 @@ public void ChangeClass(CharacterClassType classType)
             Class = classType;
             Hp = MaxHp;
             Mp = MaxMp;
-            SendPacket(GenerateTit());
-            SendPacket(GenerateStat());
-            MapInstance.SendPacket(GenerateEq());
-            MapInstance.SendPacket(this.GenerateEff(8));
+            await SendPacket(GenerateTit());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)